fishcore/ClientApp/src/components/FetchAbout.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+
+export class AboutData extends Component {
```

Comment:
This is the React "About" page, it calls the backend and is the component for the content that shows when you choose the page.